### DISCUSSION TITLE
Aplicar configuração do .editorconfig em todos os arquivos

### DISCUSSION THREAD

**Discussion by vicainelli** [frontendbr/forum#1372] [Perguntas e Respostas]
Olá, pessoas amantes do desenvolvimento web.

Venho aqui humildimente para saber se alguém já passou por isso, e , se conseguiu resolver com sucesso esse pequeno "probleminha".

To projeto aqui, e já percebi que aparecem diversos conflitos nos PR's, e a grande maioria é por formatação, por isso, queria saber se alguém sabe se é possível formatar todos os arquivos de um repositório/diretório de uma só vez com base nas configurações de um `.editorconfig`.

Agradeço desde já.

Cheers 🍺

**Comment by felipemfp** [comment]
Ultimamente, temos usado o [Prettier](https://github.com/prettier/prettier) com [quick-pretty](https://github.com/azz/pretty-quick) integrado ao hook de pre-commit do [Husky](https://github.com/typicode/husky).

**Comment by vicainelli** [comment]
> Ultimamente, temos usado o [Prettier](https://github.com/prettier/prettier) com [quick-pretty](https://github.com/azz/pretty-quick) integrado ao hook de pre-commit do [Husky](https://github.com/typicode/husky).

Nice, pensei mesmo em utilizar um hook de pre-push/pre-commmit, vou dar uma olhada com mais calma nesses links, thx. 🤘
